src/crypto/Signature.ts

```diff
@@ -3,9 +3,9 @@ import Long from 'long'
 import * as secp from '@noble/secp256k1'
 import { PublicKey, UnsignedPublicKey, SignedPublicKey } from './PublicKey'
 import { SignedPrivateKey } from './PrivateKey'
-import { utils } from 'ethers'
 import { Signer } from '../types/Signer'
-import { bytesToHex, equalBytes, hexToBytes } from './utils'
+import { bytesToHex, equalBytes, hexToBytes, splitSignature } from './utils'
+import { Hex, hashMessage } from 'viem'
```

Comment:
FWIW, we do have keccak256 available in Rust-land if you need it. If that's the primary thing you're using Viem for, maybe it makes sense to just move to that and remove the Viem dependency.

Reply:
for now, we're using `viem` for a lot more than just `keccak256`, but definitely good to know we have it in Rust if we move away from `viem` in the future.